src/UIComponents/LoadingSpinner/LoadingSpinner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import './LoadingSpinner.css'
+
+const LoadingSpinner = (props) => {
+  return (
+    <div className={`${props.asOverlay && 'loading-spinner__overlay'}`}>
+      <div className='lds-dual-ring'></div>
+    </div>
+  )
+}
+
+export default LoadingSpinner
```

Comment:
Do you prefer a props passed in convention or de-structuring? I could go either way.